perun-openapi/openapi.yml

```
@@ -5032,60 +5160,457 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/facilitiesManager/getFacilitiesWhereUserIsAdmin:
+    get:
+      tags:
+        - FacilitiesManager
+      operationId: getFacilitiesWhereUserIsAdmin
+      summary: Returns list of Facilities, where the user is an Administrator.
+      parameters:
+        - $ref: '#/components/parameters/userId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfFacilitiesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO getRichAdmins
-
-  #TODO getRichAdminsWithAttributes
+  /json/facilitiesManager/getFacilitiesByHostName:
+    get:
+      tags:
+        - FacilitiesManager
+      operationId: getFacilitiesByHostName
+      summary: Return all facilities where exists host with the specific hostname
+      parameters:
+        - { name: hostname, schema: { type: string }, in: query, required: true, description: "specific hostname"}
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfFacilitiesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO getRichAdminsWithSpecificAttributes
+  /json/facilitiesManager/getAllowedUsers:
```

```suggestion
  /json/facilitiesManager/getAllowedUsers/v-f-s:
```
There are also other versions of this method and might be in the future added to this file.

---

You should also add a TODO mentioning that there are missing some versions of this method.